cmd/cluster-operator-controller-manager/app/controllermanager.go

```diff
@@ -515,3 +517,19 @@ func startMasterController(ctx ControllerContext) (bool, error) {
 	).Run(int(ctx.Options.ConcurrentMasterSyncs), ctx.Stop)
 	return true, nil
 }
+
+func startAcceptController(ctx ControllerContext) (bool, error) {
+	if !ctx.AvailableResources[schema.GroupVersionResource{Group: "clusteroperator.openshift.io", Version: "v1alpha1", Resource: "machinesets"}] {
```

Comment:
Since the controller is syncing clusters, this should check that clusters are available rather than machinesets. Or, arguably, all of these startXXXControllers should be verifying that all of the resources that they need are available, as opposed to just the ones that the controller is directly syncing.